### DISCUSSION TITLE
not able to create pull payment with api

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#5223] [General]
I have tried through axios in my script too but it is not working.. here is a example with postman... but if i send get request to same endpoint.. it is returning all the available pull payments

{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.13",
    "title": "Unsupported Media Type",
    "status": 415,
    "traceId": "00-61e5d566185551b8bdfae36e4a55f236-d92ebdbf5bc91665-00"
}
![image](https://github.com/btcpayserver/btcpayserver/assets/137528509/e04edbf6-c2d4-461d-a1e2-c3d6dba40b01)

**Comment by ghost** [comment]
```
const headers = {
  Authorization: `token ${process.env.BTCPAY_API}`, // Example of an authentication token header
  "Content-Type": "application/json", // Example of setting the content type as JSON
};
```


Code: https://gist.github.com/akhf2004/6eb560327af2977488a3574dc44a51b8

**Reply by slydude1** [reply]
It seems to.be a scam and Google is part of it

**Reply by ghost** [reply]
what you mean ?

**Comment by ghost** [comment]
If I use Axios, it returns 422 bad request..
![image](https://github.com/btcpayserver/btcpayserver/assets/137528509/44f7ba1b-6e5e-4732-aa81-dd0930a73f03)

**Comment by mp21-org** [comment]
As mentioned in Telegram as well: You need to send a json encoded body with your POST request. Can look like this (taken from the swagger doc page):

![grafik](https://github.com/btcpayserver/btcpayserver/assets/1374810/a8929a22-1dc7-4d2e-be34-ee0e3043608c)

Also make sure to include the correct Authorization Header with your request:
"token API-KEY" (where API-KEY needs to be your API Key with sufficient privledges)
